hexrd/unitcell.py

```diff
@@ -1480,9 +1479,10 @@ def Required_C(self, C):
 
     def MakeStiffnessMatrix(self, inp_Cvals):
         if(len(inp_Cvals) != len(_StiffnessDict[self._laueGroup][0])):
-            raise IOError('number of constants entered is not correct.\
-            need a total of ', len(_StiffnessDict[self._laueGroup][0]),
-                          'independent constants')
+            x = len(_StiffnessDict[self._laueGroup][0])
+            msg = (f"number of constants entered is not correct."
```

Comment:
Technically, there is not going to be a space after this period when this error message gets printed out. But I guess it's not that big of a deal.